jobs/queue-processor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11.0
```

Comment:
same as other dockerfile comment, doesn't hurt to do it this way but having staged builds can make them smaller/quicker to build